packages/manager/modules/pci/src/projects/project/training/registries/index.js

```diff
@@ -2,16 +2,16 @@ import angular from 'angular';
 import '@uirouter/angularjs';
 import 'oclazyload';
 
-const moduleName = 'ovhManagerPciTrainingDataLazyLoading';
+const moduleName = 'ovhManagerPciTrainingRegistriesLazyLoading';
 
 angular.module(moduleName, ['ui.router', 'oc.lazyLoad']).config(
   /* @ngInject */ ($stateProvider) => {
-    $stateProvider.state('pci.projects.project.training.data.**', {
-      url: '/data',
+    $stateProvider.state('pci.projects.project.training.registries.**', {
+      url: '/registries',
```

Comment:
I'm having trouble understanding why we have a `/registries` and a `/registry`. If we have a look to how other states are formed across pci module we have 
`/instance` for listing and `/instance/add` for instance creation for example. I think we should try to follow this pattern

Reply:
We dont have /registry (only in dashboard to display a modal)